recursion.js

```diff
@@ -0,0 +1,27 @@
+let tree = {"value":100,
+    "left": {"value":90, "left":{"value":70}, "right":{"value":99 }},
+    "right":{"value":120,"left":{"value":110},"right":{"value":130}}
+};
+let tree2 = {"value":1,
+"left": {"value":2, "right":{"value":3}},
+"right":{"value":4,"left":{"value":5}}
+};
+let tree3 = {"value":125};
+
+function recursion(currentNode, depth = 0, res = []) {
```

Comment:
Also nice solution. Good approach with 'depth' as index!

Mark for this function is 20.